source/Octopus.Manager.Tentacle/TentacleConfiguration/SetupWizard/Views/ReviewAndRunScriptTabView.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Linq;
+using System.Threading;
+using System.Windows;
+using System.Windows.Controls;
+using Octopus.Manager.Tentacle.Dialogs;
+using Octopus.Manager.Tentacle.Infrastructure;
+
+namespace Octopus.Manager.Tentacle.TentacleConfiguration.SetupWizard.Views
+{
+    public partial class ReviewAndRunScriptTabView
+    {
+        public static readonly DependencyProperty SuccessMessageProperty = DependencyProperty.Register("SuccessMessage", typeof(string), typeof(ReviewAndRunScriptTabView), new PropertyMetadata(null));
+        public static readonly DependencyProperty ReadyMessageProperty = DependencyProperty.Register("ReadyMessage", typeof(string), typeof(ReviewAndRunScriptTabView), new PropertyMetadata(null));
+        public static readonly DependencyProperty TitleProperty = DependencyProperty.Register("Title", typeof(string), typeof(ReviewAndRunScriptTabView), new PropertyMetadata("Install"));
+        public static readonly DependencyProperty ExecuteButtonTextProperty = DependencyProperty.Register("ExecuteButtonText", typeof(string), typeof(ReviewAndRunScriptTabView), new PropertyMetadata("INSTALL"));
+        readonly ReviewAndRunScriptTabViewModel viewModel;
+
+        public ReviewAndRunScriptTabView(ReviewAndRunScriptTabViewModel viewModel, ContentControl additionalContent = null)
```

Comment:
I don't think the optional `additionalContent` param is ever used. Please confirm and remove if possible.